public/app/features/plugins/admin/components/PluginList.tsx

```diff
@@ -2,29 +2,42 @@ import React from 'react';
 import { css } from '@emotion/css';
 import { useStyles2 } from '@grafana/ui';
 import { GrafanaTheme2 } from '@grafana/data';
-import { CatalogPlugin } from '../types';
-import { PluginListCard } from './PluginListCard';
 import { useLocation } from 'react-router-dom';
+import { CatalogPlugin, PluginListDisplayMode } from '../types';
+import { PluginListCard } from './PluginListCard';
+import { PluginListRow } from './PluginListRow';
 
 interface Props {
   plugins: CatalogPlugin[];
+  display: PluginListDisplayMode;
 }
 
-export const PluginList = ({ plugins }: Props) => {
-  const styles = useStyles2(getStyles);
+export const PluginList = ({ plugins, display }: Props) => {
```

Comment:
**Nit:** although it's longer, I would probably call this `displayMode` instead

The types help a lot around here, but I still tend to scan the code by just looking at the variable / function / argument names first, and something like `displayMode` would tell me immediately what it is.

Reply:
Changed it in https://github.com/grafana/grafana/pull/39466/commits/4ea910a66c2541d789aedb459ada79b7e847941f.